jakarta.gradle

```diff
@@ -0,0 +1,39 @@
+def generatedSrc = "$buildDir/generated-src";
+def generatedTestSrc = "$buildDir/generated-testSrc";
+def generatedGroovyTestSrc = "$buildDir/generated-testGroovySrc";
+
+task generateSources() {
+    println "[jakarta build] replacing `javax.` with new `jakarta.`"
+    dependsOn("copyJavaSources", "copyTestJavaSources", "copyTestGroovySources")
+}
+
+task copyTestGroovySources(type: Copy) {
+    into generatedGroovyTestSrc
+    from 'src/test/groovy'
+    filter { line -> line.replaceAll("javax\\.", "jakarta\\.") }
+}
+
+task copyTestJavaSources(type: Copy) {
+    into generatedTestSrc
+    from 'src/test/java'
+    filter { line -> line.replaceAll("javax\\.", "jakarta\\.") }
+}
+
+task copyJavaSources(type: Copy) {
+    into generatedSrc
+    from 'src/main/java'
+    filter { line -> line.replaceAll("javax\\.", "jakarta\\.") }
```

Comment:
That's a bit a funny hack to make it work. I don't mind much personally as this is rather temporary, but are we sure there are no changes between `javax` and `jakarta` outside the package name ?

Reply:
I am sure for Java EE 8/Jakarta EE 9 developers there is no difference from the API level between Jakarta EE 8 and Jakarta EE 9, the biggest change is the namespace migrating to new **jakarta**.